### ISSUE TITLE
[FEATURE] Make it possible to copy a link

### ISSUE THREAD

**Issue by blueturtleai** [mastodon/mastodon-ios#725]
It would be helpful, if it would be possible to copy a link which is included in a message. Currently the only way to do this is to tap on the link, open it in the browser and copy it there.

**Comment by paulmelis** [comment]
Yes, please add this quality-of-life feature. Press-and-hold on a link in a post should bring up the copy menu, as with almost all iOS apps.
